controller/src/beerocks/master/db/node.cpp

```diff
@@ -29,6 +29,18 @@ node::node(beerocks::eType type_, const std::string &mac_)
 }
 
 namespace son {
+std::ostream &operator<<(std::ostream &os, ePersistentParamBool value)
+{
+    if (value == ePersistentParamBool::DISABLE) {
+        os << "Disabled";
+    } else if (value == ePersistentParamBool::ENABLE) {
+        os << "Enabled";
+    } else {
+        os << "Not-Configured";
+    }
+    return os;
+}
+
 std::ostream &operator<<(std::ostream &os, const node &n)
```

Comment:
```suggestion
std::ostream &operator<<(std::ostream &os, const node &node)
```
Looks weird but in the code, I prefer to see node instead of magic n.

Reply:
on this specific location I am going to leave as is - but on the new introduced APIs (in following commits) will change from `n` to `node`

Reply:
agree